inotify/src/lib.rs

```diff
@@ -559,7 +567,7 @@ pub use self::watch_mask::WatchMask;
 /// [`Inotify::rm_watch`]: struct.Inotify.html#method.rm_watch
 /// [`Event`]: struct.Event.html
 #[derive(Clone, Copy, Debug, Eq, PartialEq, Hash)]
-pub struct WatchDescriptor(c_int);
+pub struct WatchDescriptor(c_int, RawFd);
```

Comment:
I think it makes sense to turn `WatchDescriptor` into a struct with named fields at this point, but this works too.